ui/src/components/communities.tsx

```diff
@@ -76,37 +76,37 @@ export class Communities extends Component<any, CommunitiesState> {
     return (
       <div class="container">
         {this.state.loading ? (
-          <h5 class="">
+          <p class="text-center">
```

Comment:
It is centered in regards to the left column, the column with the posts in it. Because the sidebar is not loaded / rendered in yet, you don't see the full layout. 

Maybe, in the future, instead of a loading icon like this we should draw some of the layout first and show the loader in that. Or, make a full page loader. Or use those outlines as reddit does.

![Screenshot 2020-02-01 19 27 39](https://user-images.githubusercontent.com/474024/73597123-2da53980-4529-11ea-9a70-1fad5f9e8e25.png)